src/ui/containers/DependencyContainer.tsx

```diff
@@ -123,6 +124,22 @@ export const DependencyContainer: FC = ({ children }) => {
     })();
   }, []);
 
+  useEffect(() => {
```

Comment:
Above we are settings ready = true and we have all of the services available (without query to DependencyLocator), I think we should move this logic there

Reply:
Done [0db12fb](https://github.com/msupply-foundation/msupply-cold-chain/pull/323/commits/0db12fb0e923194181327b9b43684923664c858f)